here_for_change/static/js/map.js

```diff
@@ -0,0 +1,383 @@
+var municipality = {};
+var baseUrl = window.document.referrer.split("/municipalities")[0];
+var mapEl = document.querySelector("#map");
+
+if (mapEl && baseUrl) {
+  var map = L.map(mapEl);
+  var latlng = [];
+  var zoom = 8;
+  var areas = [];
+  var youAreHereLatlng = [];
+
+  let muniAreaData = [];
+  let path = window.document.referrer.split("/municipalities")[1];
+  var municipalityId = path.split("/")[1];
+  var wardId = path.split("/")[3];
+
+  fetch(`${baseUrl}/municipalities/${municipalityId}/wards/${wardId}.json`)
```

Comment:
@Hazmid the other branch has the geojson data loaded into the HTML of the page, but in this branch we're calling the JSON URL for what seems to be similar or the same data?

Reply:
The problem was that I couldn't use the html geojson data to work in the javascript file. So getting the data in json with fetch was the work around.

Reply:
@Hazmid ok, so long as we're not embedding and fetching. Choose one method to get map data.